src/components/button/Button.tsx

```diff
@@ -14,9 +14,10 @@ export type ButtonProps = {
   /** If true, substitutes the button content by an animated loader */
   loading?: boolean;
   type?: 'button' | 'reset' | 'submit';
-   /** The variant to use*/
-  variant?: 'primary' | 'secondary' | 'tertiary' | 'destructive';
   onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+   /** The variant to use*/
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'destructive' | 'primary-destructive' | 'secondary-destructive'|'tertiary-destructive' |'tertiary-accent';
```

Comment:
Could you inform that the "destructive" is deprecated please?
With @deprecated annotation, console.warn.... you can check what has been done to EmptyString, or anything if you have suggestions :)

Reply:
<img width="779" alt="Screenshot 2021-09-21 at 12 58 24" src="https://user-images.githubusercontent.com/68587873/134159147-4c1c2aa3-a3c9-4b52-a58d-23907c5b6c0c.png">